contrib/containerlab/bgpv2/auto-discovery/default-gateway/multi-homing/Makefile

```diff
@@ -0,0 +1,35 @@
+VERSION ?= v1.16.0
+CHART_DIR ?= $(shell git rev-parse --show-toplevel)/install/kubernetes/cilium
+KIND_CLUSTER_NAME ?= bgpv2-cplane-dev-mh
+VALUES_FILE ?= $(shell pwd)/values.yaml
+
+deploy:
+	kind create cluster --config cluster.yaml
+	sudo containerlab -t topo.yaml deploy
+	# create secret for bgp
+	kubectl -n kube-system create secret generic --type=string bgp-auth-secret --from-literal=password=cilium123
+	# build and load cilium images to kind
+	cd $(shell git rev-parse --show-toplevel) && sudo KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) make kind-image
```

Comment:
We don't do sudo for kind. I guess you need to add yourself to the docker group?

Reply:
ya probably. fixed